mono/metadata/sgen-mono.c

```diff
@@ -820,6 +824,20 @@ sgen_finish_concurrent_work (const char *reason, gboolean stw)
 	sgen_major_collector.finish_sweeping ();
 }
 
+void
+mono_gc_stop_world ()
```

Comment:
I don't see why we need to add this api here

Reply:
@lambdageek suggested I take this API from one of Miguel's branches, and it is a clean way to do stop the world. But I guess I do not need to actually expose it as a GC API in this PR?

Reply:
It's from https://github.com/migueldeicaza/mono/commit/4f4934f00b959911e7879216d4664b7b54184a74. It's a pretty nice thing to have if we're going to be pausing the GC for various activities

Reply:
in this case we are only suspending from within sgen so we can use directly sgen_stop/restart. If we need it some other time to expose this we can add it then.

Reply:
Remove the API.